arrow/arrow.go

```diff
@@ -26,6 +28,15 @@ func (a Arrow) Raw() []byte {
 	return a.data
 }
 
+func (a Arrow) MarshalMsgpack() ([]byte, error) {
+	return a.data, nil
+}
+
+func (a *Arrow) UnmarshalMsgpack(data []byte) error {
+	a.data = data
+	return nil
+}
```

Comment:
Please, add comments to the public API methods.

Reply:
fixed